src/routes/authentication/machine-login.ts

```diff
@@ -15,6 +15,7 @@ interface Response {
   user: User;
   environment?: Environment;
   environmentRights?: string[];
+  notificationLevel: 0 | 1 | 2 | null;
```

Comment:
Can it be made optional without any dire consequences, and the Joi schema as well?